static/app/components/feedback/list/feedbackList.tsx

```diff
@@ -78,14 +78,8 @@ export default function FeedbackList() {
     }
 
     return (
-      <CellMeasurer
-        cache={cache}
-        columnIndex={0}
-        key={key}
-        parent={parent}
-        rowIndex={index}
-      >
-        <ErrorBoundary mini>
+      <ErrorBoundary mini key={key}>
```

Comment:
moved the error boundary up outside?

Reply:
does it still work?

Reply:
seems okay for whatever terrible situation this is preventing

![image](https://github.com/user-attachments/assets/0b10867d-e593-4bef-bc20-439113417666)